test/ably/restpush_test.py

```diff
@@ -29,6 +30,12 @@ def setUpClass(cls):
         for i in range(10):
             cls.save_device()
 
+        # Register several subscriptions for later use
+        cls.channels = {'canpublish:test1': [], 'canpublish:test2': [], 'canpublish:test3': []}
+        for key, channel in zip(cls.devices, itertools.cycle(cls.channels)):
+            device = cls.devices[key]
+            cls.save_subscription(channel, device_id=device.id)
```

Comment:
Is this worth checking for a failure here? I think a silent failure can screw tests that are based on this precondition.

Reply:
I have added an assertion in https://github.com/ably/ably-python/pull/124/commits/cd9bbd2d8736df61ee4cc190d309942c1be46e51#diff-e2eb6519254bd4b1eb7b9c3357be80c6R38

Were you thinking about something different?

Reply:
That's ok. At least future you will have a hint if there's a failure due to that :)